### ISSUE TITLE
Create packages for Debian/CentOS/Fedora/

### ISSUE THREAD

**Issue by WSLUser** [neutrinolabs/pulseaudio-module-xrdp#42]
Currently only Ubuntu 18.04 Bionic has a package in launchpad. However if you install xrdp, you get a `Suggested package pulseaudio-module-xrdp` but has no source available. If I want to get audio through xrdp, I have to build it myself. That is not a acceptable solution for regular users. Users should simply install it from package manager. So please create binaries and publish them to EPEL (Fedora/CentOS) and Debian SID (since it needs to be tested by Debian team before being pushed to stable).

An alternative is to simply release the binaries on the Release page, which is easier than building yourself but has the problem of self-maintenance. There should be auto-updates using CI/CD to the repositories listed above.

**Comment by metalefty** [comment]
> If I want to get audio through xrdp, I have to build it myself. That is not a acceptable solution for regular users. Users should simply install it from package manager.

I completely agree but it's up to each distro team right now. They should able to do that much better than us.

I'm also a FreeBSD developer and quite familiar with creating FreeBSD packages, so I create and maintain [PulseAudio module of xrdp](https://www.freshports.org/audio/pulseaudio-module-xrdp). To create packages for other OS, we xrdp community need such packaging experts like me of FreeBSD.

Due to human resources constraints, we release source only.  We always welcome any kinds of contributions.

**Comment by docop** [comment]
i did install all the command, in a root ssh login.

**Comment by docop** [comment]
details: 

with like the 3 last command
it said : libtool: warning : relinking 'module-mmkdb-evdev.la ... and other module. like for around 40-50..
---
near the end : 
`with :    cd pulseaudio-module-xrdp
[root@c999945779-cloudpro-905586277 pulseaudio-module-xrdp]# ./bootstrap && ./configure PULSE_DIR=~/rpmbuild/BUILD/pulseaudio-10.0
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
aclocal: warning: couldn't open directory 'm4': No such file or directory
autoreconf: configure.ac: tracing
autoreconf: configure.ac: creating directory build-aux
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, `build-aux'.
libtoolize: copying file `build-aux/ltmain.sh'
libtoolize: putting macros in `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
`

**Comment by DRRDietrich** [comment]
For Fedora/CentOS/RHEL, the problem has been listed in the Red Hat Bug Tracker since October 2019 and was assigned to Itamar Reis Peixoto.
[https://bugzilla.redhat.com/show_bug.cgi?id=1757616](https://bugzilla.redhat.com/show_bug.cgi?id=1757616)

**Comment by cpcn** [comment]
For now can  pulseaudio-module-xrdp be built with pulseaudio ( from Debian Buster packages) already installed?

**Comment by metalefty** [comment]
pulseaudio-module-xrdp requires PulseAudio source to build. Distro packages usually provides only source.

**Comment by lnee94** [comment]
i made a script to auto make a deb for the wired pulse audio headers https://github.com/lnee94/pulseaudio-headers-xrdb

**Comment by DollarCat** [comment]
I am operating on CentOS 7.7 system.
When I execute the command: yumdownloader -- source pulsaudio, it prompts' no source rpm found for pulsaudio-10.0-5. el7.x86_64 '. What can I do to continue？Any help is appreciated.

**Comment by matt335672** [comment]
CentOS 7 is EOL, so it's not generally mirrored in the usual places anymore.

Have you updated your yum repos to point to https://vault.centos.org/ ? That's the only place you'll be able to get source RPMs from now that I'm aware of.
